configure.ac

```diff
@@ -251,28 +251,34 @@ AC_TYPE_INT16_T
 AC_TYPE_INT32_T
 AC_TYPE_INT64_T
 
+PKG_CHECK_MODULES([libqb], [libqb])
+
 if test "x$enable_man" = "xyes"; then
 	AC_ARG_VAR([DOXYGEN], [override doxygen executable])
 	AC_CHECK_PROGS([DOXYGEN], [doxygen], [no])
 	if test "x$DOXYGEN" = xno; then
 		AC_MSG_ERROR(["Doxygen command not found"])
 	fi
-	# required by doxyxml to build man pages dynamically
-	# Don't let AC_PROC_CC (invoked by AX_PROG_CC_FOR_BUILD) replace
-	# undefined CFLAGS_FOR_BUILD with -g -O2, overriding our special OPT_CFLAGS.
-	: ${CFLAGS_FOR_BUILD=""}
-	AX_PROG_CC_FOR_BUILD
```

Comment:
Maybe drop the now-unused `m4/ax_prog_cc_for_build.m4`?

Reply:
good catch, i´ll kill it tomorrow.

Reply:
done